analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -279,9 +279,9 @@ public void Verify_ErrorBehavior()
         {
             var builder = WithSnippetCS("undefined");
             builder.Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>()
-                .WithMessage("CSharp7: Unexpected build error [CS0116]: A namespace cannot directly contain members such as fields, methods or statements on line 1");
+                .WithMessage("CSharp7: Unexpected build error [CS8805]: Program using top-level statements must be an executable. on line 1");
```

Comment:
They changed the error message that will appear in this case. As long as error happens, the UT is fine.